packages/core/src/components/forms/unsafe_controls.tsx

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
these components are unnecessary. if the polyfill is doing its job right, why would anyone need the unsafe versions? we should not expand Blueprint's API surface area by adding these components which duplicate so much code.

Reply:
yes, i don't like them either. this is what i figured you meant with https://github.com/palantir/blueprint/pull/3702#discussion_r325780179. i'll remove them